enos/enos.py

```diff
@@ -187,7 +187,12 @@ def install_os(env=None, **kwargs):
         'neutron_external_address':   pop_ip(env),
         'network_interface':          env['eths'][NETWORK_IFACE],
         'kolla_internal_vip_address': env['config']['vip'],
-        'neutron_external_interface': env['eths'][EXTERNAL_IFACE]
+        'neutron_external_interface': env['eths'][EXTERNAL_IFACE],
+        # Specific variables for the Ansible plugin
+        'influx_vip':                 env['config']['influx_vip'],
+        'kolla_ref':                  env['config']['kolla_ref'],
+        'enable_monitoring':
```

Comment:
IMO we should add `enable_monitoring` only if the key exist. We shouldn't choose a default value here since it will be set by ansible automatically when reading the `groups_vars/all.yml`